packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -110,118 +122,186 @@ describe('Facet Test Suite', () => {
+          Expect.numberOfSelectedCheckboxValues(0);
+          Expect.displayMoreMatchesFound(false);
+          Expect.displayNoMatchesFound(true);
+          Expect.noMatchesFoundContainsQuery(query);
+          Expect.displaySearchClearButton(true);
+        });
+      });
+    });
 
-            Expect.logFacetSelect(defaultField, 0);
+    describe('show more/less values', () => {
+      describe('when facet has no more values', () => {
+        function showAllValues() {
+          visitFacetPage({
+            field: defaultField,
+            label: defaultLabel,
+            numberOfValues: 1000,
           });
+          cy.wait(InterceptAliases.Search);
+        }
+
+        describe('verify rendering', () => {
+          before(showAllValues);
+
+          Expect.displayShowMoreButton(false);
+          Expect.displayShowLessButton(false);
         });
+      });
 
-        describe('when searching for a value that returns a single result', () => {
-          describe('verify rendering', () => {
-            before(searchForSingleValue);
+      describe('when clicking show more values', () => {
+        const smallNumberOfValues = 2;
 
-            Expect.numberOfIdleCheckboxValues(1);
-            Expect.numberOfSelectedCheckboxValues(0);
-            Expect.displayMoreMatchesFound(false);
-            Expect.displayNoMatchesFound(false);
-            Expect.displaySearchClearButton(true);
+        function showMoreValues() {
+          visitFacetPage({
+            field: defaultField,
+            label: defaultLabel,
+            numberOfValues: smallNumberOfValues,
           });
-        });
+          cy.wait(InterceptAliases.Search);
+          FacetSelectors.showMoreButton().click();
+          cy.wait(InterceptAliases.Search);
+        }
 
-        describe('when searching for a value that returns no results', () => {
-          const query = 'this facet value does not exist';
+        describe('verify rendering', () => {
+          before(showMoreValues);
+
+          Expect.numberOfValues(smallNumberOfValues * 2);
+        });
 
-          function searchForInvalidValue() {
-            setupWithValues();
-            FacetSelectors.searchInput().type(query);
+        describe('when clicking show more button again', () => {
+          function showMoreValuesAgain() {
+            showMoreValues();
+            FacetSelectors.showMoreButton().click();
+            cy.wait(InterceptAliases.Search);
           }
 
           describe('verify rendering', () => {
-            before(searchForInvalidValue);
+            before(showMoreValuesAgain);
 
-            Expect.numberOfIdleCheckboxValues(0);
-            Expect.numberOfSelectedCheckboxValues(0);
-            Expect.displayMoreMatchesFound(false);
-            Expect.displayNoMatchesFound(true);
-            Expect.noMatchesFoundContainsQuery(query);
-            Expect.displaySearchClearButton(true);
+            Expect.numberOfValues(smallNumberOfValues * 3);
```

Comment:
I think we should add test to make sure the FacetValues is sorted after click on 'ShowMore' button, as that's an important feature

Reply:
Same as [this comment](https://github.com/coveo/ui-kit/pull/1244#discussion_r716792788).

Reply:
like our discussion, please add the validation for the facetValues order when clicking on ShowMore/Less buttons, thanks!